assets/scss/components/_sidebar.scss

```diff
@@ -0,0 +1,32 @@
+.c-sidebar__box {
```

Comment:
@mgbelegu, I suggest using the sidebar-bar for the component/block here. So, the class structure would be: `c-sidebar-box`, `c-sidebar-box__list`, and so on.